baseplate/secrets/store.py

```diff
@@ -255,6 +240,33 @@ def get_versioned(self, path):
             next=next_value and _decode_secret(path, encoding, next_value),
         )
 
+    def get_vault_url(self):
+        """Return the URL for accessing Vault directly.
+
+        :rtype: :py:class:`str`
+
+        .. seealso:: The :py:mod:`baseplate.context.hvac` module provides
+            integration with HVAC, a Vault client.
+
+        """
+        self._load_if_needed()
+        return self._data["vault"]["url"]
+
+    def get_vault_token(self):
+        """Return a Vault authentication token.
+
+        The token will have policies attached based on the current EC2 server's
+        Vault role. This is only necessary if talking directly to Vault.
+
+        :rtype: :py:class:`str`
+
+        .. seealso:: The :py:mod:`baseplate.context.hvac` module provides
+            integration with HVAC, a Vault client.
+
+        """
+        self._load_if_needed()
+        return self._data["vault"]["token"]
```

Comment:
I think that's fine... the only thing it could break this way is dev environments which can be fixed pretty easily